roles/build_edge_installer/tasks/main.yml

```diff
@@ -1,11 +1,16 @@
 ---
+- name: Set facts
+  ansible.builtin.set_fact:
+    builder_blueprint_temp: '{{ builder_blueprint_name }}'
```

Comment:
Why is this necessary?

Reply:
That was the only way I could get the var shared or passed to the builder role. I tried using `defaults_from` but I think its only used when you want to change to a different file within the same role.

Is there another way of doing this? perhaps a cleaner approach? :smile:

Reply:
commenting here for posterity, we discussed this on Matrix and decided to merge the two roles and have the installer be built based on parameters to the `builder` role ... though we probably want to move that to a different PR away from the kickstart vars